pycodestyle.py

```diff
@@ -2105,7 +2105,16 @@ def read_config(options, args, arglist, parser):
 
     parent = tail = args and os.path.abspath(os.path.commonprefix(args))
     while tail:
-        if config.read(os.path.join(parent, fn) for fn in PROJECT_CONFIG):
+        try:
+            files_read = config.read(os.path.join(parent, fn)
+                                     for fn in PROJECT_CONFIG)
+        except configparser.ParsingError:
+            if options.verbose:
+                print('failed to read local configuration files due to a '
+                      'parsing error')
```

Comment:
I don't have strong opinions on that. It's not exactly the most useful error message and it will certainly make the verbose output less readable for most people.